### ISSUE TITLE
Code about training with gradient accumulation

### ISSUE THREAD

**Issue by qtz980805** [MiZhenxing/GBi-Net#9]
Sorry to bother again.
In recent days, I am trying to train your network with gradient accumulation. However, my implementation still doesn‘t work，i.e., the training loss does not decrease. 
I would be very appreciate if you could help provide the code about training with gradient accumulation.
Thanks and look forward to your reply!
